rules/DowngradePhp70/Rector/Expression/DowngradeUnnecessarilyParenthesizedExpressionRector.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\DowngradePhp70\Expression;
```

Comment:
PHPStan is little overzealous:

  - '#Class "Rector\\DowngradePhp70\\Rector\\Expression\\DowngradeUnnecessarilyParenthesizedExpressionRector" has invalid namespace category "Expression"\. Pick one of\: "ArrayDimFetch", PropertyFetch", MethodCall", StaticPropertyFetch", StaticCall", FuncCall"#'

It forces me to pick a single one when I match all of them.